modules/tinymce/changelog.txt

```diff
@@ -1,31 +1,31 @@
 Version 5.5.0 (TBD)
     Added a TypeScript declaration file to the bundle output for TinyMCE core #TINY-3785
-    Added new `table_column_resizing` setting to control how the table is affected when resizing columns using the resize bars #TINY-6001
-    Added ability to load TinyMCE in inline mode within a ShadowRoot #TINY-6128
+    Added new `table_column_resizing` setting to control how columns are resized when resizing tables using the resize bars #TINY-6001
```

Comment:
There's a lot of `resizing` in this sentence. Is that okay?

Reply:
it is okay, just not pleasant. 
Should `bars` be `handles` ?

Reply:
Could `how columns are resized when resizing tables using the resize bars` maybe be `how table columns are resized when using the resize bars`?

Reply:
no, they are actual `bars` (horiztonal/vertical lines in the table). It could maybe be `resize bar handles`? That however reads strangely to me.

Reply:
accepted @metricjs 's suggestion, leaving "bar handles" as just "bars"